src/main/java/com/epam/test_generator/entities/Project.java

```diff
@@ -117,14 +117,25 @@ public void addUser(User user) {
 
     @Override
     public void addSuit(Suit suit) {
-        suit.setRowNumber(this.suits
+        suit.setRowNumber(lastRowNumber() + 1);
+        this.suits.add(suit);
+    }
+
+    @Override
+    public void addSuits(List<Suit> suits) {
+        Integer lastRow = lastRowNumber() + 1;
```

Comment:
Hello,
You have this code twice
`lastRowNumber() + 1`
It is better to extract this to a simple private method.

What problem could be?
You can change it sometime in one place and forget change it in the other.

In addition to that it is better to avoid '+1's and '-1' in methods because you need to think everytime, why I have +1 here?
It is better to have well named method, so your code will be more self-descriptive.